tests/integration/integration-test.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-set -euo pipefail
+set -euxo pipefail
```

Comment:
not a big fan of wildly making fixes at completely different points. Feel free to make a comment on #944